atmo/jobs/models.py

```diff
@@ -280,6 +281,11 @@ def save(self, *args, **kwargs):
         if self.expired_date and self.end_date and self.end_date > timezone.now():
             self.expired_date = None
         super().save(*args, **kwargs)
+        # Remove the cached latest run to this objects will requery it.
```

Comment:
This was needed since the cached_property was storing state when a new spark run was created using factory_boy in the tests.